pkgs/applications/misc/usql/default.nix

```diff
@@ -1,32 +1,44 @@
 { lib
+, stdenv
 , fetchFromGitHub
 , buildGoModule
 , unixODBC
 , icu
+, nix-update-script
+, testers
+, usql
 }:
 
-buildGoModule rec {
+let
+  vendorHash =
+    if stdenv.isDarwin then
+      "sha256-aq2RtuYSNh7wc7rtEKxUv8FvWyTlxKF8/tvgxPVjbe4="
+    else
+      "sha256-9fAdfPqsIFFdfH6dBD/xSb8wmAszwxKxS7ovsHAR0cY=";
```

Comment:
Did you try proxyVendor?

Reply:
Hello @SuperSandro2000

Indeed, looks like enabling useProxyVendor is a resolution.

I also rebased against master and squashed my commits.

Thanks for the review!